modules/index-service/src/main/java/org/opencastproject/index/service/impl/IndexServiceImpl.java

```diff
@@ -2079,4 +2144,13 @@ public boolean isWorkflowActive(String workflowState) {
             || WorkflowState.PAUSED.toString().equals(workflowState);
   }
 
+  private void updateEngagePublication(MediaPackage mediaPackage) throws PublicationException {
+    if (!autoRepublish) {
+      return;
+    }
+
+  engagePublicationService.publishSync(mediaPackage, null, PUBLISH_STRATEGY_MERGE,
```

Comment:
My thinking was that for the use case proposed here these are basically set, i.e. you don't want `checkAvailability` and `strategy` needs to be `"merge"`. It was further inspired/cemented by the `republish-metadata` workflow. That assumption might be wrong, though, and obviously adding config for those options isn't a problem if people want it.

Reply:
Well `merge` is a problem because of #5963. Not for ACL and dublincore/episode, but next people want to republish preview images and then you have leaks.